Scripts/Tracking/Velocity/VelocityEstimator.cs

```diff
@@ -225,6 +257,7 @@ protected virtual void EstimateAngularVelocity(float factor)
                 int sampleIndex = currentSampleCount % angularVelocitySamples.Length;

```

Comment:
(I actually wanted to comment on the `if` line.)

Same as above: Negate to `== 0` and make it an early return right at the start of the method, otherwise we're doing all this math without using it.